components/support/nimbus-fml/src/intermediate_representation.rs

```diff
@@ -954,6 +967,26 @@ pub mod unit_tests {
         Ok(())
     }
 
+    #[test]
+    fn validate_allow_coenrollment() -> Result<()> {
+        let mut fm = get_simple_homescreen_feature();
+        fm.feature_defs.push(FeatureDef::new(
+            "some_def",
+            "test doc",
+            vec![PropDef {
+                name: "prop name".into(),
+                doc: "prop doc".into(),
+                typ: TypeRef::EnumMap(
+                    Box::new(TypeRef::Enum("SectionId".into())),
+                    Box::new(TypeRef::String),
+                ),
+                default: json!(null),
+            }],
+            true,
+        ));
+        fm.validate_manifest()
```

Comment:
Then check if `fm.get_coenrolling_feature_ids()` works, and returns `vec!["some_def".to_string()]`.